kymatio/scattering1d/backend/torch_skcuda_backend.py

```diff
@@ -60,8 +60,8 @@ def get_blocks(self, N):
     def __call__(self, x):
         if not x.is_cuda and self.backend=='skcuda':
             raise TypeError('Use the torch backend (without skcuda) for CPU tensors.')
-
-        out = torch.empty_like(x[..., :1])
+        
+        out = torch.empty(x.shape[:-1] + (1,), device=x.device, layout=x.layout, dtype=x.dtype)
```

Comment:
I have to admit I was happier with the `empty_like` because it's more concise. But as long as it works, I'm OK